components/checkbox/checkbox.js

```diff
@@ -8,16 +8,24 @@ function Checkbox(props) {
   const {
     checked,
     children,
+    dataAttrs = {},
     disabled,
     name,
     onChange,
+    ...otherProps
```

Comment:
In my opinion, it is better to remove `otherProps` because it provides too much power and leads to components misuse and unstable behavior. 

Btw, If you have `otherProps` you don't need ANY other props, so you can remove them as well :D